src/main/java/uk/gov/hmcts/cmc/claimstore/documents/BulkPrintService.java

```diff
@@ -140,9 +141,34 @@ public void printPdf(Claim claim, List<Printable> documents) {
         );
     }
 
+    @Override
+    public void printGeneralLetterPdf(Claim claim, List<Printable> documents) {
```

Comment:
printGeneralLetterPdf is same implementation of printLetterPdf except for the wrapGeneralLetterInMap. its better to have it as same method and just creating the map of parameters to be done either as supplier or function.